objects/obj_creation_popup/main.js

```
@@ -0,0 +1,719 @@
+      {/* Toolbar */}
+      <div className="flex flex-wrap items-center gap-2 p-3 bg-white/80 backdrop-blur border-b">
+        <ToolButton active={tool===TOOLS.FREE} onClick={()=>setTool(TOOLS.FREE)} label="Freehand" icon="✏️"/>
+        <ToolButton active={tool===TOOLS.POLY} onClick={()=>setTool(TOOLS.POLY)} label="Polygon" icon="🔺"/>
+        <ToolButton active={tool===TOOLS.MEASURE} onClick={()=>setTool(TOOLS.MEASURE)} label="Measure" icon="📏"/>
+        <ToolButton active={tool===TOOLS.CALIB} onClick={()=>setTool(TOOLS.CALIB)} label="Calibrate" icon="🎚️"/>
+        <ToolButton active={tool===TOOLS.PAN} onClick={()=>setTool(TOOLS.PAN)} label="Pan/Zoom" icon="✋"/>
+
+        <div className="mx-4 h-6 w-px bg-gray-200"/>
+
+        <label className="text-sm text-gray-600">Stroke</label>
+        <input type="color" value={strokeColor} onChange={(e)=>setStrokeColor(e.target.value)} className="w-10 h-9 rounded"/>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)